todo-app/src/Components/DeleteButton.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+export default class DeleteButton extends Component {
+    handelDisabledButton = (selected) => {
+        if (selected === false) {
```

Comment:
You can simply do `return selected === false` instead of these `if else` statements.